test/KBaseReport_server_test.pl

```diff
@@ -13,7 +13,7 @@
 my $config = new Config::Simple($config_file)->get_block('KBaseReport');
 my $ws_url = $config->{"workspace-url"};
 my $ws_name = undef;
-my $ws_client = new Bio::KBase::workspace::Client($ws_url,token => $token);
+my $ws_client = Workspace::WorkspaceClient->new($ws_url,token => $token);
 my $auth_token = Bio::KBase::AuthToken->new(token => $token, ignore_authrc => 1);
```

Comment:
I think this line can not work properly. Why don't you refactor token validation similar to https://github.com/kbase/kb_sdk/blob/auth2/src/java/us/kbase/templates/module_test_perl_client.vm.properties#L22 ?